Release-Notes/Release3.md

```diff
@@ -137,6 +153,8 @@ updates for R3 at the end of April.
 
 ## List of known issues & restrictions in R3
 
+* Distributed Virtual Routing (DVR) is not officially supported by us, not tested and not recommended.
```

Comment:
I would not use the phrase "supported by us" - but instead "supported by OSISM".
I actually had the same thought, when I read that change in the OSISM's changelog and wanted to suggest the same wording there.